src/app/tables/artifacts-table/artifacts-table.component.spec.ts

```diff
@@ -153,4 +154,97 @@ describe('ArtifactsTableComponent', () => {
+    expect(tableHeadersText[0]).toBe('Advisories');
+    expect(tableHeadersText[1]).toBe('Completion Time');
+    expect(tableHeadersText[2]).toBe('ID');
+    expect(tableHeadersText[3]).toBe('Name');
+    expect(tableHeadersText[4]).toBe('Owner');
+    expect(tableHeadersText[5]).toBe('Release');
+    expect(tableHeadersText[6]).toBe('Version');
+
+    const activeColumnsText = fixture.debugElement.query(By.css('.estuary-table-header__columns-text')).nativeElement;
+    expect(activeColumnsText.textContent.trim()).toBe('7 of 18 columns selected');
+
+    // Ensure the actual table content is correct.
+    const rows = fixture.debugElement.queryAll(By.css('.estuary-table tbody > tr'));
+    expect(rows.length).toBe(2);
+    const rowOneColumns = rows[0].nativeElement.children;
+    expect(rowOneColumns[0].textContent.trim()).toBe('RHSA-2018:1234-07  RHSA-2018:1235-07');
+    // Test that each advisory has its own correct link
+    let idLink = rowOneColumns[0].children[0];
+    expect(idLink.tagName).toBe('A');
+    expect(idLink.textContent.trim()).toBe('RHSA-2018:1234-07');
+    expect(idLink.href).toBe(`${errataUrl}33491`);
+    idLink = rowOneColumns[0].children[1];
+    expect(idLink.tagName).toBe('A');
+    expect(idLink.textContent.trim()).toBe('RHSA-2018:1235-07');
+    expect(idLink.href).toBe(`${errataUrl}33492`);
+    expect(rowOneColumns[1].textContent.trim()).toBe('November 30, 2018, 02:55:47 UTC');
+    expect(rowOneColumns[2].textContent.trim()).toBe('806970');
+    expect(rowOneColumns[3].textContent.trim()).toBe('openstack-container');
+    expect(rowOneColumns[4].textContent.trim()).toBe('freshmaker');
+    expect(rowOneColumns[5].textContent.trim()).toBe('67.1543534138');
+    expect(rowOneColumns[6].textContent.trim()).toBe('13.0');
+
+    const rowTwoColumns = rows[1].nativeElement.children;
+    expect(rowTwoColumns[0].textContent.trim()).toBe('RHSA-2018:1234-07  RHSA-2018:1235-07');
+    // Test that each advisory has its own correct link
+    idLink = rowTwoColumns[0].children[0];
+    expect(idLink.tagName).toBe('A');
+    expect(idLink.textContent.trim()).toBe('RHSA-2018:1234-07');
+    expect(idLink.href).toBe(`${errataUrl}33481`);
+    idLink = rowTwoColumns[0].children[1];
+    expect(idLink.tagName).toBe('A');
+    expect(idLink.textContent.trim()).toBe('RHSA-2018:1235-07');
+    expect(idLink.href).toBe(`${errataUrl}33482`);
+    expect(rowTwoColumns[1].textContent.trim()).toBe('November 30, 2018, 02:55:47 UTC');
+    expect(rowTwoColumns[2].textContent.trim()).toBe('806971');
+    expect(rowTwoColumns[3].textContent.trim()).toBe('openstack-container');
+    expect(rowTwoColumns[4].textContent.trim()).toBe('freshmaker');
+    expect(rowTwoColumns[5].textContent.trim()).toBe('67.1543534138');
+    expect(rowTwoColumns[6].textContent.trim()).toBe('13.0');
+
+    // Click the dropdown button so that the menu appears
+    const dropdownButton = fixture.debugElement.query(
```

Comment:
@sarah256 a lot of this test duplicates the previous test. Could you slim down this test to just the things that are different?